types/artifacts.d.ts

```diff
@@ -29,6 +29,8 @@ declare global {
       NetworkUserAgent: string;
       /** The benchmark index that indicates rough device class. */
       BenchmarkIndex: number;
+      /** The application manifest if one was fetched. */
+      AppManifest: Artifacts.Manifest | null;
```

Comment:
to me this makes me question if it's actually related to an appcache manifest :) If we want to be more precise, how do you feel about `WebAppManifest`?

Reply:
someone should tell driver and the devtools protocol then 😛 

that wfm 👍